app/src/main/java/org/jellyfin/androidtv/ui/itemhandling/BaseRowItem.java

```diff
@@ -493,13 +495,20 @@ public Drawable getBadgeImage(Context context) {
     public void refresh(final EmptyResponse outerResponse) {
         switch (type) {
             case BaseItem:
-                apiClient.getValue().GetItemAsync(getItemId(), KoinJavaComponent.<UserRepository>get(UserRepository.class).getCurrentUser().getValue().getId().toString(), new Response<BaseItemDto>() {
-                    @Override
-                    public void onResponse(BaseItemDto response) {
-                        baseItem = response;
-                        outerResponse.onResponse();
-                    }
-                });
+                try {
+                    apiClient.getValue().GetItemAsync(getItemId(), KoinJavaComponent.<UserRepository>get(UserRepository.class).getCurrentUser().getValue().getId().toString(), new Response<BaseItemDto>() {
+                        @Override
+                        public void onResponse(BaseItemDto response) {
+                            baseItem = response;
+                            outerResponse.onResponse();
+                        }
+                    });
+                } catch (Exception e) {
```

Comment:
Exception it too generic, catching just NullPointerException should be enough for the linked issues.